samples/dotnet/kernel-syntax-examples/RepoUtils/ConsoleLogger.cs

```diff
@@ -20,13 +20,17 @@ private static ILoggerFactory LogBuilder()
     {
         return LoggerFactory.Create(builder =>
         {
-            // builder.SetMinimumLevel(LogLevel.Warning);
+            builder.SetMinimumLevel(LogLevel.Warning);
+
             // builder.AddFilter("Microsoft", LogLevel.Trace);
             // builder.AddFilter("Microsoft", LogLevel.Debug);
             // builder.AddFilter("Microsoft", LogLevel.Information);
-            builder.AddFilter("Microsoft", LogLevel.Warning);
+            // builder.AddFilter("Microsoft", LogLevel.Warning);
             // builder.AddFilter("Microsoft", LogLevel.Error);
+
+            builder.AddFilter("Microsoft", LogLevel.Warning);
             builder.AddFilter("System", LogLevel.Warning);
+
             builder.AddConsole();
```

Comment:
Can you update Example08_RetryHandler or set min level for `Program` to information?